pyigm/cgm/ingest_lit.py

```diff
@@ -118,18 +122,82 @@ def p11():
     p11.to_json_tarball(out_file)
 
 
+def ingest_burchett16():
+    """ Ingest Burchett+16
+    """
+    # Virial matching
+    b16_vir_file = resource_filename('pyigm', 'data/CGM/z0/Burchett16_HI_virselect_sfr.fits')
```

Comment:
In testing this, I noticed that the coordinates for the galaxies in the file I have you were incorrect (crazy impact parameters in ion_tbl).  I have revised versions for you.

Reply:
ok